politeiawww/config.go

```diff
@@ -61,7 +62,11 @@ const (
 	// }
 	dust = 60300
 
+	// Currently available modes to run politeia, by default piwww, is used.
 	politeiaWWWMode = "piwww"
+	cmsWWWMode      = "cmswww"
```

Comment:
Needs to update the config description:
https://github.com/decred/politeia/blob/199749d8e0d4d8f32538fb5f915a3cadbb6c83d7/politeiawww/config.go#L127